### DISCUSSION TITLE
Exit codes

### DISCUSSION THREAD

**Discussion by mihaistate** [esbmc/esbmc#1475] [Q&A]
Hello! I couldn't find out what ESBMC exit code: -9 and -6 do. Could anyone, please, explain me what these errors mean?

**Comment by lucasccordeiro** [comment]
@rafaelsamenezes / @fbrausse : could you please reply to this question?

**Reply by fbrausse** [reply]
I have no idea. ESBMC returns positive 6 when the GOTO conversion fails here: https://github.com/esbmc/esbmc/blob/ac48dbcef4860f83d4aba6f2f2a6f4b3153d6762/src/esbmc/esbmc_parseoptions.cpp#L496-L497

How negative exit codes happen I don't know. @mihaistate Do you have an example input/command line that produces these exit codes?
